data/France/National_Assembly/sources/instructions.json

```diff
@@ -76,6 +76,21 @@
         "from": "election-wikidata",
         "base": "Q22160703"
       }
+    },
+    {
+      "file": "morph/twitter.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/french-assembly-twitter-list-scraper",
+        "query": "SELECT * FROM data"
+      },
+      "type": "person",
+      "source": "https://twitter.com/AssembleeNat/lists/les-députés",
+      "merge": {
+        "incoming_field": "name",
+        "existing_field": "name__fr",
```

Comment:
I'm still picking up a few you've missed, and I suspect you're missing some here because this is only going to match people who explicitly have a 'name__fr' field, which some won't. In general this should just be checking against 'name' unless you definitely want names in a different language (which usually means names in a different writing system, like Russian or Arabic etc)